server/views/pages/overview/partials/overviewTab/overviewTabPreInduction.njk

```diff
@@ -22,7 +22,7 @@
         html: createInductionBanner
       }) }}
 
-      {% include '../_goalsList.njk' %}
+      {% include './_goalsList.njk' %}
```

Comment:
I moved this back to only being shared between the overview screens. My plan is to move the sorting out of the service layer and allow the view to decide the order when iterating